woocommerce/woocommerce-android
# [Coupons] Coupon Details Menu Functionality

Part of: #5533

Please do not merge until #6165 is merged



### Description


This adds the "Copy" and "Share" functionality in the Coupon Details screen.

### Testing instructions


1. Start app, go to Menu, then Coupons.
2. Select a coupon, this will load the Coupon Details screen.
3. Check and ensure the top app bar is shown.
4. Tap the triple-dot menu on top right and make sure two menu items are shown: "Copy Coupon Code" and "Share Coupon".
5. Test the "Copy Coupon Code" functionality.
6. Test the "Share Coupon" functionality.
